### PR TITLE
make the fast parabolic MF parametrization dynamic

### PR DESCRIPTION
**Pull request by namapane**
This is necessary as this parametrization is used as fast map in tracking in offline and HLT.

### CODE REVIEW

**Comment by slava77** [comment]
@cmsbuild please test

File: MagneticField/ParametrizedEngine/plugins/AutoParametrizedMagneticFieldProducer.cc [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    version = "Uniform";
+    parameters.push_back(0);
+  }
+
+  else if (version=="Parabolic"){
+    parameters.push_back(3.8114);       //c1
+    parameters.push_back(-3.94991e-06); //b0
+    parameters.push_back(7.53701e-06);  //b1
+    parameters.push_back(2.43878e-11);   //a
+    if (cnc!=18268){ // Linear scaling for B!= 3.8T; note that just c1, b0 and b1 have to be scaled to get linear scaling
```
Comment (slava77) [inline MagneticField/ParametrizedEngine/plugins/AutoParametrizedMagneticFieldProducer.cc]: Does this actually scale, especially for low currents?

Comment (namapane) [inline MagneticField/ParametrizedEngine/plugins/AutoParametrizedMagneticFieldProducer.cc]: To very good approxymation,  inside the solenoid, it scales with 
current; please note that I don't scale with the current itself but with 
the closest nominal working point. Nominal WPs are 2, 3, 3.5 and 3.8T; 
maps are available only for these, and likewise I made this fast 
parametrization stick to the same.

At some point we could re-fit the 2, 3, 3.5T parameters, but that does 
not seem to be so fundamental, if we have short runs at B other than 
3.8T we can live with the decent approx of linear scaling.

Cheers
	Nicola

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/8992	e5574c0
- changes in code look OK
- jenkins tests run and comparisons show no differences as expected

**Comment by slava77** [comment]
@cmsbuild is the bot asleep? please wake it up
I've sent my "+1" 20 mins ago

**Pull request merged**